SQA_AT/src/test/java/ui/SecondTaskTest.java

```diff
@@ -30,7 +35,7 @@ public void registerUser() {
         String email = "john_smit2@gmail.com";
```

Comment:
would be better to create a user on test class level (after u will fix builder) and reuse this user in tests
e.g.     User testUser = new User(firstName,lastName , email etc);